server/core/basic_cluster.go

```diff
@@ -16,6 +16,9 @@ package core
 import (
 	"sync"
 
+	"github.com/pingcap/log"
+	"go.uber.org/zap"
+
```

Comment:
remove this line.

Reply:
I need to output logs in the following code.
```
func (bc *BasicCluster) CheckAndPutRegion(region *RegionInfo) []*RegionInfo {
	origin, err := bc.PreCheckPutRegion(region)
	if err != nil {
		log.Warn("region is stale", zap.Error(err), zap.Stringer("origin", origin.GetMeta()))
		// return the state region to delete.
		return []*RegionInfo{region}
	}
	return bc.PutRegion(region)
}
```

Reply:
the empty line.

Reply:
done